doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -92,3 +92,130 @@ Talk to the Lab Owners if special hardware or topology requirements are needed,
 **Lab Quantities and Duration of Lab Use**
 
 There is no limit to the number of labs which can be secured, or the duration.  The limiting factor is resource available.  Lab Owners reserve the right to reclaim underutilized labs and reassign to other teams where there is demand for labs.  It is important that the Lab User have a schedule and plan in place to utilize the labs to retain the lab for testing.
+
+<a name="4.5"></a>
+## 4.5 OPNFV Pharos Project
```

Comment:
@markbeierl 4.2 specifies the CNTT Lab hardware requirements while 4.5.2 specifies the Pharos specifications. Will there be an attempt to reconcile the Pharos and CNTT lab specifications in light of the objectives to have CNTT specifications compliant NFVI?